pages/common/git-verify-commit.md

```diff
@@ -0,0 +1,17 @@
+# git verify-commit
+
+> Check for GPG verification of commits.
+> If no commits are verified, nothing will be printed, regardless of options specified.
+> More information: <https://git-scm.com/docs/git-verify-commit>.
+
+- Check commits for a GPG signature:
+
+`git verify-commit {{commit_hash}} {{commit_hash}}`
```

Comment:
@bl-ue do you think this is fine or should we use `{{commit_hash1}} {{commit_hash2}}`? 

@CleanMachine1 the second commit hash should also be specified in the other two examples

Reply:
no, the user can infer that

Reply:
@marchersimon I'd prefer numbering them each.

@CleanMachine1 is the second hash required in the other examples?  We make it a point _not_ to have the user infer much, unless it's quite obvious (and here I'd say it _isn't_.)

Reply:
But the user shouldn't have to. We generally try to keep every example as independent as possible.

Reply:
I would feel better if it was {{optional_commit_hash}}

Reply:
How about `{{commit_hashes}}`?

Reply:
No, infact that makes it worse since it implies that multiple hashes are required

Reply:
@CleanMachine1 have you seen @bl-ue's comment [above](https://github.com/tldr-pages/tldr/pull/5946#discussion_r631241098)? I'm sure it wasn't there yesterday. Probably another GitHub bug.

Reply:
I didn't see it either.

Reply:
GH doesn't automatically load new comments in thread, very unfortunately.  You **have** to reload the page.  That comment was there yesterday, maybe you guys didn't reload so never saw it? 🤔

Reply:
I reload every time I get a notification..

Reply:
if we just add {{optional_hash(s')}} that would be sufficient I feel

Reply:
Oh, so it can be 1 or more? not restricted to 2?

Reply:
It can be as many as you like

Reply:
Ah, okay.  "optional" implies _0_ or more, though, but at least one is required, correct?

Reply:
Yes, without 1 you get this

```
$ git verify-commit
usage: git verify-commit [-v | --verbose] <commit>...

    -v, --verbose         print commit contents
    --raw                 print raw gpg status output

```

Reply:
the command should be `git verify-commit {{commit_hash}} {{optional_commit_hash(s')}}

Reply:
We actually have a special syntax just for cases like this:
```
git verify-commit {{commit_hash1 commit_hash2 ...}}
```

Reply:
Im going to add a commit and you tell me what you think.

Reply:
that implies you need more than 1

Reply:
> the command should be `git verify-commit {{commit_hash}} {{optional_commit_hash(s')}}`

While technically correct, it just looks overly complicetad. Using `{{commit_hash1 commit_hash2 ...}}` is way cleaner and it's nothing tldr-specific. When you specify a gerenal list you can also write `item1, item2, item3, ...`, but everyone knows, that a list can also contain only one element.

Reply:
https://github.com/tldr-pages/tldr/pull/5946/commits/52a31ad6edd8b4ceb37d3b4f9ce5dc2b440c56a7 fixes the issues however makes it look a bit uglier, however it doesn't make the user take a guess at anything

Reply:
We used to recommend something similar to `Check a space-separated list of commit hashes` but I think @bl-ue replaced this format with another proposal, but `optional_commit_hash(s')`is not friendly based on our format accross several pages.

Reply:
Give a suggestion quote and I'll approve

Reply:
This conversation is on an outdated comment, which means that suggestions are disabled.

Reply:
Add to new